K2Bridge.Tests.End2End/TestElasticClient.cs

```diff
@@ -120,7 +120,7 @@ public async Task<JToken> MSearch(string indexName, string jsonQueryFile, bool v
 
             using var request = new HttpRequestMessage(HttpMethod.Post, "_msearch");
             var payload = new StringBuilder();
-            payload.AppendLine($"{{\"index\":\"{indexName}\"}}");
+            payload.AppendLine(FormattableString.Invariant($"{{\"index\":\"{indexName}\"}}"));
```

Comment:
Why is it needed? Why only here?

Reply:
This is the only place the compiler issued a warning